_aggregations/metric/average.md

```diff
@@ -9,9 +9,20 @@ redirect_from:
 
 # Average aggregations
 
-The `avg` metric is a single-value metric aggregations that returns the average value of a field.
+The `avg` metric is a single-value metric that returns the average value of a field.
 
-The following example calculates the average of the `taxful_total_price` field:
+## Parameters
+
+The `avg` aggregation takes the following parameters: 
+
+| Parameter | Required/Optional | Data type       | Description |
+| :--       | :--               | :--            | :--         |
+| `field`   | Required          | Numeric         | The field for which the average is computed    |
+| `missing` | Optional          | Numeric         | Value to assign missing instances of the field |
```

Comment:
```suggestion
| `missing` | Optional          | Numeric         | Value to assign missing instances of the field. |
```

Reply:
If you don't provide the `missing` field, are the missing values ignored?

Reply:
@kolchfa-aws Yes, the average is computed without them. I give an example and a counterexample.

Reply:
```suggestion
| `missing` | Optional          | Float         | The value to assign to missing instances of the field. By default, `avg` omits missing values from the calculation. |
```